runtime/compiler/control/J9Options.cpp

```diff
@@ -2072,12 +2084,18 @@ J9::Options::setupJITServerOptions()
       {
       TR::PersistentInfo *persistentInfo = compInfo->getPersistentInfo();
       if (persistentInfo->getRemoteCompilationMode() == JITServer::SERVER)
+         {
          TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer, "JITServer Server Mode. Port: %d. Connection Timeout %ums",
                persistentInfo->getJITServerPort(), persistentInfo->getSocketTimeout());
+         }
       else if (persistentInfo->getRemoteCompilationMode() == JITServer::CLIENT)
+         {
          TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer, "JITServer Client Mode. Server address: %s port: %d. Connection Timeout %ums",
                persistentInfo->getJITServerAddress().c_str(), persistentInfo->getJITServerPort(),
                persistentInfo->getSocketTimeout());
+         TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer,"Identifier for current client JVM: %llu\n", 
```

Comment:
We really ought to be using `PRIu64` from `<inttypes.h>` instead of `llu` in places like this.